src/styles/layout/index.md.njk

```diff
@@ -123,3 +123,20 @@ The desktop specific classes also allow you to set the width of the tablet break
 ### Nested grids
 
 {{ example({group: "styles", item: "layout", example: "nested", html: true, open: true, size: "m"}) }}
+
+### Display override classes
```

Comment:
This should be a h2 so `## Display override classes` (which will also place it in the nav)